bin/fillTestData/Messages.js

```diff
@@ -171,7 +149,7 @@ function seedThreads() {
               function addMessage(depth, userTo, userFrom) {
                 const message = new Message();
 
-                message.created = addDays(Date.now(), -depth + 1);
+                message.created = util.addDays(Date.now(), -depth + 1);
```

Comment:
Just FYI, we also have `moment` available which has some convenience methods. Moment.js isn't great at frontend (because of bloat) but kinda fine at scripts like this.

Fine using the simple utility as well though. :-) It might just behave oddly on the last day of the month?

Reply:
Edit: oh I see you just moved the existing utility to another file. Never mind!